documents/process/release.md

```diff
@@ -32,38 +32,49 @@ This is a checklist of things that should be done in the weeks leading to the re
-* Get this reviewed and checked in before continuing
-* Publish each `icu_*` crate
-  * Use `cargo workspaces publish --from-git` to automatically publish the crates in the correct order if you would like
-  * Add `icu4x-release` group as owners to each new component you're publishing
-    * `cargo owner -a github:unicode-org:icu4x-release`
-* [Tag the Release](https://github.com/unicode-org/icu4x/releases) with the text drafted above
-* Update and publish FFI packages
-  * Dart
-    * update version in `ffi/dart/pubspec.yaml`
-    * update the artifacts tag in `ffi/dart/lib/src/hook_helpers/version.dart` to the tag created above
-    * get this checked in, then `cd ffi/dart && dart pub publish`
-  * JS
-    * update version in `icu4x/ffi/npm/package.json`
-    * get this checked in, then `cd ffi/npm && npm publish`
-* Create a branch named `release/x.y` including the release tag and FFI commits and push it to the upstream
-* Update the website
-  * In the `icu4x-docs` repo, run `node tools/github-to-astro.ts` and follow the instructions
-* Announce the release to public
-  * (All releases) Send an email to [icu4x-announce](https://groups.google.com/u/0/a/unicode.org/g/icu4x-announce)
-  * (All releases) Blog post on Unicode blog
-  * (Major releases only) Submit to This Week In Rust
+* [ ] Land the changelog (see below)
+* [ ] Go through the prerelease checklist again, ensuring that no problems were reintroduced in the PRs that landed since the opening of the checklist. (Things like doc prettification will likely need to be rerun!)
+* [ ] Prepare a PR with updated versions
+  * [ ] Remove all `-dev` prelease tags from `Cargo.toml`s
+  * [ ] Update all ICU4X crate versions
+    * [ ] Update the workspace version to the new version
+    * [ ] Some `icu_*` crates do not follow the ICU4X versioning scheme: `icu_codepointtrie_builder`, `icu_pattern`, and `icu_experimental`. Be sure to give them an appropriate version based on the changelog. Major releases are always paired with a `0.x.0` release of `icu_experimental`.
+    * [ ] Find all ICU4X component/provider crates that have an overridden version in their `Cargo.toml` and reset it to `version.workspace = true`.
+    * [ ] Update all ICU4X `~` dependencies in the toplevel `Cargo.toml`'s `workspace.dependencies`.
+    * [ ] Update `icu_locale_core`, `icu_provider`, and `icu_pattern`'s non-`~` dependency in `Cargo.toml`'s `workspace.dependencies`
+      * Note: These are in a different section and easy to miss!
+  * [ ] Update all relevant utils versions
+    * [ ] The changelog should already mention the new versions of each util crate that needs to be published. Use those, and double-check that they are accurate.
+    * [ ] For utils that have had substantiative changes (new APIs, etc), update them in `workspace.dependencies`. When unsure, just update.
+* [ ] Get this PR reviewed and checked in.
```

Comment:
if you have "prepare PR" step directly followed by a "merge this PR" step, it can be a single step